_includes/app.html

```diff
@@ -1,8 +1,8 @@
 <div class="col-lg-4 col-md-6 mb-3 mt-3 d-flex flex-column">
-  <a href="{{ app.url }}" class="d-flex flex-column bg-white rounded-1 box-shadow border text-inherit no-underline flex-1 overflow-hidden" style="height:100%">
+  <a href="{{ app.url }}" class="d-flex flex-column bg-white rounded-1 box-shadow border text-inherit no-underline flex-1" style="height:100%">
     <h3 class="h4 px-3 pt-3">{{ app.title }}</h3>
     <p class="text-gray lh-condensed px-3 pb-3">{{ app.description }}</p>
-    <div class="text-gray-light pl-3 pr-2 py-2 bg-gray-light border-top" style="margin-top: auto;">
+    <div class="text-gray-light pl-3 pr-2 py-2 bg-gray-light border-top rounded-1" style="margin-top: auto; border-top-left-radius: 0 !important; border-top-right-radius: 0 !important;">
```

Comment:
Would you mind rewriting this a little so that this doesn't have `rounded-1`, and instead the style attribute will look like:

```
style="border-bottom-left-radius: 3px; border-bottom-right-radius: 3px;"
```

This eliminates the `!important`s which makes it a little cleaner - still far from ideal, but it'll work.